src/declarations/stencil-public-compiler.ts

```diff
@@ -1557,7 +1557,7 @@ export interface CompilerSystemRemoveFileResults {
 
 export interface CompilerSystemWriteFileResults {
   path: string;
-  error: any;
+  error: Error;
```

Comment:
Can we revert this + add a TODO(STENCIL-NUM) comment here? Technically, I think this would be a breaking change to the public Node API since we're narrowing this type.

Reply:
Good catch, I reverted the change.